lib/ansible/modules/cloud/vmware/vmware_host_ntp.py

```diff
@@ -16,57 +17,79 @@
 DOCUMENTATION = r'''
 ---
 module: vmware_host_ntp
-short_description: Manage NTP configurations about an ESXi host
+short_description: Manage NTP server configuration of an ESXi host
 description:
-- This module can be used to manage NTP configuration information about an ESXi host.
+- This module can be used to configure, add or remove NTP servers from an ESXi host.
+- If C(state) is not given, the NTP servers will be configured in the exact sequence.
 - User can specify an ESXi hostname or Cluster name. In case of cluster name, all ESXi hosts are updated.
 version_added: '2.5'
 author:
 - Abhijeet Kasurde (@Akasurde)
+- Christian Kotte (@ckotte)
 notes:
 - Tested on vSphere 6.5
 requirements:
 - python >= 2.6
 - PyVmomi
 options:
-  cluster_name:
-    description:
-    - Name of the cluster.
-    - NTP settings are applied to every ESXi host system in the given cluster.
-    - If C(esxi_hostname) is not given, this parameter is required.
   esxi_hostname:
     description:
-    - ESXi hostname.
-    - NTP settings are applied to this ESXi host system.
-    - If C(cluster_name) is not given, this parameter is required.
+    - Name of the host system to work with.
+    - This parameter is required if C(cluster_name) is not specified.
+    type: str
+  cluster_name:
+    description:
+    - Name of the cluster from which all host systems will be used.
+    - This parameter is required if C(esxi_hostname) is not specified.
+    type: str
   ntp_servers:
     description:
-    - "IP or FQDN of NTP server/s."
+    - "IP or FQDN of NTP server(s)."
     - This accepts a list of NTP servers. For multiple servers, please look at the examples.
+    type: dict
     required: True
   state:
     description:
-    - "present: Add NTP server/s, if it specified server/s are absent else do nothing."
-    - "absent: Remove NTP server/s, if specified server/s are present else do nothing."
-    default: present
+    - "present: Add NTP server(s), if specified server(s) are absent else do nothing."
+    - "absent: Remove NTP server(s), if specified server(s) are present else do nothing."
```

Comment:
Could you please add a note here stating - if state is not specified then NTP servers will be overwrite with user value. 


With current documentation user has to dig into the code to get this information.

Reply:
Ok. Done.